addons/website/views/snippets/s_unveil.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_unveil" name="Unveil">
+    <section class="s_unveil pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="13">
+                <div class="o_grid_item g-height-3 g-col-lg-12 col-lg-12" style="grid-area: 1 / 1 / 3 / 13;">
```

Comment:
You forgot to set the z-index :slightly_smiling_face:  => all grid items must have a z-index so the options are consistent (note that it should begin at 1 so set it to 1 here and increase the other ones below by one)